### ISSUE TITLE
Header & Footer Control ??

### ISSUE THREAD

**Issue by ravishakya** [WordPress/gutenberg#13031]
I am a Theme Developer in WP.org. In past, I use customizer for header, footer and other parts of the site. With the release of Gutenberg and WordPress 5.0, I am thinking of shifting it to the blocks. My main concern is some sections will be in blocks and some will be in customizer( Header and Footer ). Is there a way or is there a plan to control the header and footer from the blocks. The widgets are coming into blocks, so will the control of header and footer will also be added ??

**Comment by Soean** [comment]
This is a topic for Gutenberg phase 2, so yes 2019. Details: https://make.wordpress.org/core/2018/12/08/gutenberg-phase-2/

**Comment by KaiBuskirk** [comment]
should have been a Phaze One Beta block..

**Comment by paaljoachim** [comment]
The thought here is that we should keep the consistency in selecting layout elements (building the layout). By adding the blocks needed to create the layout we want. So perhaps the customizer becomes the area that has most focus on the global areas. While Gutenberg has a stronger focus on per page/post layout. There will of course be overlapping of them both.

![global-blocks](https://user-images.githubusercontent.com/5323259/50556339-1c5f7d00-0cd8-11e9-848c-8b5cbe7a9207.jpg)


A Global block area inside the Block Selector.
Contains Body (Global body/HTML), Header, Sidebar and Footer.
Could be enough with just Body and Sidebar. But I added the Header and Footer to make it easier for the user to understand.

----------

Here is an idea for selecting a Global Body. It adds a thin border around the layout (perhaps not feasible but use it as inspiration..:) and has a toolbar top left. I did it this way as it would show that the user has selected a Global Body block that will be seen on specific pages/posts etc.
I also added a conditionals area in the Advanced section of the options.

![global-body-block](https://user-images.githubusercontent.com/5323259/50556350-35682e00-0cd8-11e9-9136-e140182be260.jpg)

I am going to link in this issue as well (Ideas for Global Block Settings issue):
https://github.com/WordPress/gutenberg/issues/1224
